packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -45,6 +45,10 @@ export type PersonalizeMiddlewareConfig = MiddlewareBaseConfig & {
    * Configuration for your Sitecore CDP endpoint
    */
   cdpConfig: CdpServiceConfig;
+  /**
+   * Flag to set the enablePersonalizeCookie setting of cloud sdk personalize; if omitted, defaults to false
+   */
+  enablePersonalizeCookie?: boolean;
```

Comment:
what happens when this cookie is absent? Is it only the A/B that doesn't work, or would personalize rules based on visitor ID be affected too?
Overall perhaps it makes sense to always set this cookie, unless personalization middleware is disabled in the seprate setting.

Reply:
i.e. could this cookie affect "The visitor has visited page name(s) during the current visit" and other similar rules?

Reply:
I was thinking to provide a link to the csdk documentation for more details, because we are only just passing this through..?
You mean that opt out can be done via the disable function?